pkg/reconciler/route/resources/names/names.go

```diff
@@ -38,5 +41,5 @@ func Ingress(route kmeta.Accessor) string {
 // Certificate returns the name for the Certificate
 // child resource for the given Route.
 func Certificate(route kmeta.Accessor) string {
-	return "route-" + string(route.GetUID())
+	return fmt.Sprintf("route-%d", adler32.Checksum([]byte(route.GetUID())))
```

Comment:
```suggestion
	return "route-"+strconv.Itoa(int(adler32.Checksum([]byte(route.GetUID()))))
```
the cast might be redundant :)

Reply:
`adler32.Checksum` returns a `uint32` and `strconv.Itoa` wants an `int`.
https://play.golang.org/p/YzWY7ujL3dO

Reply:
So I thought, so we need the cast 📦